### PR TITLE
FEA-78- Fill shortcuts with weather data

### PR DESCRIPTION
**Pull request by klaudia-blazyczek-cc**

### CODE REVIEW

**Review by JakubLukaszCC — APPROVED** [review]

**Review by lukasz-konatowski-cc — APPROVED** [review]

**Review by adrian-konopnicki-CC — APPROVED** [review]

**Review by kamil-borecki-cc — APPROVED** [review]

File: src/components/CityWeatherShortcut/CityWeatherShortcut.styled.tsx [hunk 1]
```diff
@@ -48,6 +47,14 @@ const City = styled.p`
   font-size: 20px;
   line-height: 24px;
   text-align: center;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  max-width: 80%;
+  white-space: nowrap;
```
Comment (JakubLukaszCC) [inline src/components/CityWeatherShortcut/CityWeatherShortcut.styled.tsx:53]: it should appear on `City` element

Comment (klaudia-blazyczek-cc) [reply src/components/CityWeatherShortcut/CityWeatherShortcut.styled.tsx:53]: It is applied to City component

**Review by JakubLukaszCC — APPROVED** [review]

File: src/components/CityWeatherShortcut/CityWeatherShortcut.styled.tsx (hunk 1 above)
Comment (JakubLukaszCC) [reply src/components/CityWeatherShortcut/CityWeatherShortcut.styled.tsx:53]: right, my bad github displayed it weirdly

**Review by AgataLudwiczynska-CC — APPROVED** [review]

**Review by adrian-konopnicki-CC — APPROVED** [review]

File: src/components/CityWeatherShortcut/CityWeatherShortcut.tsx [hunk 2]
```diff
@@ -21,14 +22,16 @@ const CityWeatherShortcut = ({ cityName }: Props) => {
...
     return (
-      <Container data-testid={CityWeatherShortcutTestIds.Widget}>
-        <Icon src={condition.icon} alt="weather image" />
-        <Temperature>
-          <Value>{roundedTemperature}</Value>
-          <TemperatureSign>&deg;C</TemperatureSign>
-        </Temperature>
-        <City>{data.location.name}</City>
-      </Container>
+      <Tooltip title={data.location.name}>
```
Comment (bednarskimichal) [inline src/components/CityWeatherShortcut/CityWeatherShortcut.tsx]: please use destructing for `name` and reuse it as a name of city

Comment (klaudia-blazyczek-cc) [reply src/components/CityWeatherShortcut/CityWeatherShortcut.tsx]: @bednarskimichal done

**Review by bednarskimichal — APPROVED** [review]

**Pull request merged**